app/docker/docker-compose.selenium.yml

```diff
@@ -10,7 +10,8 @@ services:
       com.centurylinklabs.watchtower.enable: 'true'
     shm_size: '2gb'
     environment:
-      START_XVFB: 'true'
+      - "START_XVFB=true"
```

Comment:
all the rest use this method and I see no reason to change to the array version, START_XVFB looks much more complex now and it will not highlight well for `dab apps config`.